From f2bef1f0da60b32ccc5e4778ec9db3c8ef563590 Mon Sep 17 00:00:00 2001
From: Pirmin Vogel <vogelpi@lowrisc.org>
Date: Tue, 11 Feb 2020 17:27:12 +0100
Subject: [PATCH 3/5] [prim] Less secret change

This change is less secret, we are going to merge it back upstream.

Signed-off-by: Pirmin Vogel <vogelpi@lowrisc.org>

diff --git a/hw/ip/prim/rtl/prim_ram_2p_async_adv.sv b/hw/ip/prim/rtl/prim_ram_2p_async_adv.sv
index 978775a..4b4e994 100644
--- a/hw/ip/prim/rtl/prim_ram_2p_async_adv.sv
+++ b/hw/ip/prim/rtl/prim_ram_2p_async_adv.sv
@@ -69,6 +69,8 @@ module prim_ram_2p_async_adv #(
                              (Width <= 120) ? 8 : 8 ;
   localparam int TotalWidth = Width + ParWidth;
 
+  // Less secret change
+
   logic                  a_req_q,    a_req_d ;
   logic                  a_write_q,  a_write_d ;
   logic [SramAw-1:0]     a_addr_q,   a_addr_d ;
-- 
2.17.1

